src/externs/select2.js

```diff
@@ -27,6 +27,15 @@ select2.Event.prototype.params;
  */
 jQuery.prototype.select2 = function(opt_params) {};
 
+/** @type {?} */
```

Comment:
question mark needs to be avoided as much as possible. Arguments would be necessary here to explain why you want the compiler to not check this code.